File: Data Structures & Algorithms/remove-element/submission-2.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        vector<int> ans;

        int x=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==val){
                x++;
            }
        }
        for(int i=0;i<nums.size();i++){
            if(nums[i]==val){
                x--;
                continue;
            }
            ans.push_back(nums[i]);
        }
        nums=ans;
        return nums.size()-x;
    }
};

File: Data Structures & Algorithms/string-encode-and-decode/submission-2.py
class Solution:

    def encode(self, strs: List[str]) -> str:
        ans=""
        if len(strs)==0:
            return ans
        
        for st in strs:
            ans+=str(len(st))+'#'+st
        
        return ans

    def decode(self, s: str) -> List[str]:
        ans=[]

        if len(s)==0:
            return ans

        i=0
        while i<len(s):
            j=i
            while s[j]!='#':
                j+=1
            length=int(s[i:j])
            word = s[j+1:j+1+length]
            i=j+1+length
            ans.append(word)
        
        return ans
